monai/apps/nnunet/nnunet_bundle.py

```diff
@@ -0,0 +1,448 @@
+            checkpoint = torch.load(
+                join(Path(model_training_output_dir).parent, "nnunet_checkpoint.pth"), map_location=torch.device("cpu")
+            )
+            monai_checkpoint = torch.load(join(model_training_output_dir, model_name), map_location=torch.device("cpu"))
+            if i == 0:
+                trainer_name = checkpoint["trainer_name"]
+                configuration_name = checkpoint["init_args"]["configuration"]
+                inference_allowed_mirroring_axes = (
+                    checkpoint["inference_allowed_mirroring_axes"]
+                    if "inference_allowed_mirroring_axes" in checkpoint.keys()
+                    else None
+                )
+
+            if "network_weights" in monai_checkpoint.keys():
+                parameters.append(monai_checkpoint["network_weights"])
+            else:
+                parameters.append(monai_checkpoint)
+
+        configuration_manager = plans_manager.get_configuration(configuration_name)
+        # restore network
+        import nnunetv2
+        from nnunetv2.utilities.find_class_by_name import recursive_find_python_class
+        from nnunetv2.utilities.label_handling.label_handling import determine_num_input_channels
+
+        num_input_channels = determine_num_input_channels(plans_manager, configuration_manager, dataset_json)
+        trainer_class = recursive_find_python_class(
+            join(nnunetv2.__path__[0], "training", "nnUNetTrainer"), trainer_name, "nnunetv2.training.nnUNetTrainer"
+        )
+        if trainer_class is None:
+            raise RuntimeError(
+                f"Unable to locate trainer class {trainer_name} in nnunetv2.training.nnUNetTrainer. "
+                f"Please place it there (in any .py file)!"
+            )
+        network = trainer_class.build_network_architecture(
+            configuration_manager.network_arch_class_name,
+            configuration_manager.network_arch_init_kwargs,
+            configuration_manager.network_arch_init_kwargs_req_import,
+            num_input_channels,
+            plans_manager.get_label_manager(dataset_json).num_segmentation_heads,
+            enable_deep_supervision=False,
+        )
+
+        predictor.plans_manager = plans_manager
+        predictor.configuration_manager = configuration_manager
+        predictor.list_of_parameters = parameters
+        predictor.network = network
+        predictor.dataset_json = dataset_json
+        predictor.trainer_name = trainer_name
+        predictor.allowed_mirroring_axes = inference_allowed_mirroring_axes
+        predictor.label_manager = plans_manager.get_label_manager(dataset_json)
+        if (
+            ("nnUNet_compile" in os.environ.keys())
+            and (os.environ["nnUNet_compile"].lower() in ("true", "1", "t"))
+            # and not isinstance(predictor.network, OptimizedModule)
+        ):
+            print("Using torch.compile")
+            # predictor.network = torch.compile(self.network)
```

Comment:
It seems this line need to be uncommented otherwise will not using `torch.compile`